install/migrations/update_10.0.x_to_11.0.0/configs.php

```diff
@@ -44,10 +44,17 @@
     'set_followup_tech' => '0',
     'set_solution_tech' => '0',
     'is_demo_dashboards' => '0',
+    'planned_task_state' => Planning::TODO,
 ]);
 $migration->addField('glpi_users', 'toast_location', 'string');
 
 $migration->addField('glpi_users', 'set_followup_tech', 'tinyint DEFAULT NULL');
 $migration->addField('glpi_users', 'set_solution_tech', 'tinyint DEFAULT NULL');
+$migration->addField(
+    'glpi_users',
+    'planned_task_state',
+    'int DEFAULT NULL',
+    ['value' => Planning::TODO]
```

Comment:
`value` is for the default value (and it is alrready defined by `DEFAULT NULL`).
```suggestion
    ['update' => 1]
```